### DISCUSSION TITLE
GLB model not loading animation properly in scene viewer

### DISCUSSION THREAD

**Discussion by pzyu** [google/model-viewer#3646] [Q&A]
### Description

This is a new issue I have encountered only today in scene viewer. My model is supposed to have an animation pose, but it does not reflect in scene viewer. 

I did not update the model-viewer version, so I'm not entirely sure if the changes are on Google's scene viewer or whether the issue lies somewhere else.

#### Live Demo
When previewing with https://arvr.google.com/scene-viewer-preview, the model falls back to A-pose and does not play my animation pose.
![123123](https://user-images.githubusercontent.com/18459487/87061355-3a4b5d80-c23e-11ea-891a-912a40d12062.PNG)

Previewing with https://gltf-viewer.donmccurdy.com/ gives the expected result
It also shows up fine in model-viewer, before jumping into AR mode
![222](https://user-images.githubusercontent.com/18459487/87061358-3b7c8a80-c23e-11ea-87e2-ac72cb2e98b3.PNG)

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions

- model-viewer: v1.0.0

**Comment by elalish** [comment]
@tpsiaki Looks like a SceneViewer regression?
